lib/util/include/hse_util/arch.h

```diff
@@ -15,10 +15,14 @@
  */
 #define HSE_RA_PAGES_MAX        ((128 * 1024) / PAGE_SIZE)
 
+/* SMP_CACHE_BYTES is deprecated in favor of HSE_L1D_LINESIZE.
+ */
 #if (LEVEL1_DCACHE_LINESIZE > 64)
+#define HSE_L1D_LINESIZE        (LEVEL1_DCACHE_LINESIZE)
 #define SMP_CACHE_BYTES         (LEVEL1_DCACHE_LINESIZE)
```

Comment:
Can we remove it if its deprecated?

Reply:
Now that Nabeel has checked in I'll go ahead and convert the remaining uses and get rid of it, yay!

Reply:
I noticed in one of your C11 atomics PRs you left the old atomic_t and atomic64_t types around. Maybe now is also a good opportunity to remove those.

Reply:
Yep, Nabeel is going to fix all the ones in wal, then I'll take care of whatever is left...

Reply:
SMP_CACHE_BYTES is eradicated by my second commit!